sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/samples/Sample4_ExportAndImport.md

```diff
@@ -0,0 +1,121 @@
+# Export and Import Question Answering Projects
+
+This sample demonstrates how to export and import Question Answering projects. To get started, you'll need to create a Question Answering service endpoint and an API key. See the [README](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/README.md) for links and instructions.
+
+To export, import, or perform any other authoring actions for Question Answering projects, you need to first create a `QuestionAnsweringProjectsClient` using an endpoint and API key. These can be stored in an environment variable, configuration setting, or any way that works for your application.
+
+```C# Snippet:QuestionAnsweringProjectsClient_Create
+Uri endpoint = new Uri("https://myaccount.api.cognitive.microsoft.com");
+AzureKeyCredential credential = new AzureKeyCredential("{api-key}");
+
+QuestionAnsweringProjectsClient client = new QuestionAnsweringProjectsClient(endpoint, credential);
+```
+
+Once you have created a client, you can call synchronous or asynchronous methods.
+
+## Synchronous
+
+### Exporting a Project
+
+To export a Question Answering project, you will need to set your project name and choose an export format before calling c as shown below:
+
+```C# Snippet:QuestionAnsweringProjectsClient_ExportProject
+string exportFormat = "json"; // can also be tsv or excel
+bool waitForCompletion = true;
```

Comment:
Same suggestion here (and check for all instances) about just using named params if you really want to emphasize what passing `true` would mean below. Just seems like an anti-pattern.